src/components-examples/material/checkbox/checkbox-overview/checkbox-overview-example.html

```diff
@@ -5,21 +5,20 @@
 
 <section class="example-section">
   <span class="example-list-section">
-    <mat-checkbox class="example-margin"
-                  [checked]="allComplete"
-                  [color]="task.color"
-                  [indeterminate]="someComplete()"
-                  (change)="setAll($event.checked)">
-      {{task.name}}
+    <mat-checkbox
+      class="example-margin"
+      [checked]="task().completed"
+      [indeterminate]="partiallyComplete()"
+      (change)="update($event.checked)"
+    >
+      {{task().name}}
     </mat-checkbox>
   </span>
   <span class="example-list-section">
     <ul>
-      @for (subtask of task.subtasks; track subtask) {
+      @for (subtask of task().subtasks; track subtask; let i = $index) {
```

Comment:
We can also use `$index` directly without having to alias it. Not a big deal to leave it as is though.